### ISSUE TITLE
how to set out_grads in mx.mod.backward(out_grads) for a multi-output network

### ISSUE THREAD

**Issue by nicklhy** [apache/incubator-mxnet#7879]
I have a network similar to the code below:
```
x = mx.sym.Variable('x')
y = mx.sym.Variable('y')
w = mx.sym.Variable('w')
b = mx.sym.Variable('b')

xw = mx.sym.dot(x, w) # a feature map that I need to check in each iteration
loss = mx.sym.LinearRegressionOutput(data=xw+b, label=y) # a loss layer

net = mx.sym.Group([xw, loss])
mod = mx.mod.Module(symbol=net, ...)
# bind, init_params, init_optimizer ...
```
This network is able to backward without a problem if I didn't add `xw` as an output layer. But due to some specific reasons, I need to get it during the training phase.
How should I set `out_grads` when calling `mod.backward(out_grads)` to make the network calculating gradients exactly as the occasion that `xw` is not an output layer?

**Comment by KeyKy** [comment]
@nicklhy  Hi hongye! I use this~~
```
xw = mx.symbol.MakeLoss(mx.symbol.stop_gradient(data=xw))
net = mx.sym.Group([xw, loss])

```

**Comment by kaonashi-tyc** [comment]
Can confirm @KeyKy 's recipe works.
